dts/bindings/sensor/adi,adxl375-common.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 Aaron Chan
+# SPDX-License-Identifier: Apache-2.0
+
+include: sensor-device.yaml
+
+properties:
+  odr:
```

Comment:
You could implement this as a string property and use DT_INST_ENUM_IDX.
That way the user can use the required value directly in their devicetree config, e.g:
https://github.com/zephyrproject-rtos/zephyr/blob/f0b975032a68c2da38c34f01b836edb5e9e62b12/dts/bindings/pinctrl/ambiq%2Capollo4-pinctrl.yaml#L76-L82